Swift/ConsoleLogger.swift

```diff
@@ -20,6 +20,33 @@
 import Foundation
 
 /// Console logger for writing log messages to the system console.
+
+/// Log domain options that can be enabled in the console logger.
+public struct LogDomains: OptionSet {
+    /// Raw value.
+    public let rawValue: Int
+    
+    /// Constructor with the raw value.
+    public init(rawValue: Int) {
+        self.rawValue = rawValue
+    }
+    
+    /// All domains.
+    public static let all = LogDomains(rawValue: Int(LogDomain.all.rawValue))
+    
+    /// Database domain.
+    public static let database = LogDomains(rawValue: Int(LogDomain.database.rawValue))
```

Comment:
are we doing this? 
```suggestion
    public static let database = LogDomains(rawValue: 1 << 0)
```

Reply:
I prefer to keep it that way so that we have the raw value defined in one place. The value needs to be matched with Objc’s value as well.

Reply:
LogDomain.database has done that.